xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/java/org/xwiki/officeimporter/server/script/OfficeServerScriptService.java

```diff
@@ -59,6 +60,18 @@
      */
     private static final String ERROR_PRIVILEGES = "Inadequate privileges.";
 
+    /**
+     * Prefix of the translation keys for server states.
+     */
+    private static final String TRANSLATION_KEY_SERVER_STATE_PREFIX =
+        "xe.officeimporter.openoffice.serverstate.";
```

Comment:
Looks like the prefix does not match the newly introduced keys.

Reply:
Indeed. It's should be better now.